src/main/java/ToDo.java

```diff
@@ -1,11 +1,13 @@
-public class ToDo extends Task{
+public class ToDo extends Task {
```

Comment:
For this todo part, you can try to see the partial solution of level 4 and you will find that there's a better way to return the information of each task